bin/dotd

```diff
@@ -129,7 +130,9 @@ EOS
 user_name = ask_for_name
 
 should_i "morning DTT" do
-  open_pull_request 'test', 'staging', 'DTT (Staging > Test)'
+  GitHub.create_pull_request(
+    base: 'test', head: 'staging', title: 'DTT (Staging > Test)'
+  )
```

Comment:
Key behavior change here: `open_pull_request` pops open a browser window to the pull request creation page with things autofilled, so the DOTD is two clicks from merging.  `GitHub.create_pull_request` takes care of the first click automatically (creating the pull request) but you're not auto-merging it here and I believe we're no longer opening the browser to make that easy to do.  So can we do one of:

* Use `open_url` to pop open the newly created pull request
* (My preference) Just auto-merge the PR here too

In fact, (and this is probably a future PR,) I'd love for the DOTD script here to also check the `#developers` topic and give you a heads-up if DTT is no for any reason (like you need to check on the robo-DTT).  Then it should give you a chance to loop back to this step once you've cleared any blocking issues :grin:.